app/controllers/users/sessions_controller.rb

```diff
@@ -91,13 +93,21 @@ def cache_active_profile
     end
 
     def user_signed_in_and_not_locked_out?(user)
```

Comment:
I didn't change this behavior. This method existed before this PR. Can we revisit later if necessary?